script.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Check if `git` exists in the current env.
+if ! command -v git &> /dev/null; then
+    echo "git could not be found"
+    exit 2
+fi
+
+ERRORS=`git log --grep='^\w+(\((\w|-)+\))?!?: .+$' --invert-grep -E --pretty=format:%s | awk 'END {print NR}'`
+echo "$ERRORS"
+
+if [[ -n $GITHUB_ACTIONS && $GITHUB_ACTIONS == true ]]; then
```

Comment:
Usage of `[[` is superfluous: you do not need pattern matching, `test` or `[` is enough.

Reply:
Thank you, I'll use the `[` notation